### PR TITLE
FIX: remove trigger path in CI build

### PR DESCRIPTION
**Pull request by stijnmoreels**
Since the CI build verifies the YAML builds; we should trigger on changes in the YAML builds.

I removed all trigger paths entirely to be safe.

### CODE REVIEW

File: build/ci-build.yml
Comment (tomkerkhove) [inline build/ci-build.yml]:
```suggestion
      - master
    paths:	
    include:	
      - build/ci-build.yml
      - src/*
```

**Review by tomkerkhove — DISMISSED** [review]

**Comment by stijnmoreels** [comment]
Hmm, I don't think I could get this building green since it will use the current build solution YAML template resource and not the one I'm changing here in this PR.

**Comment by tomkerkhove** [comment]
Ok, I'll let you merge it then so you can check and fix the CI in case of issues after merging, ok?

**Review by tomkerkhove — APPROVED** [review]

**Comment by stijnmoreels** [comment]
> Ok, I'll let you merge it then so you can check and fix the CI in case of issues after merging, ok?

Ok, but I think it can take an additional PR's each time to verify if it succeeds or not; maybe there's a better way.

**Comment by tomkerkhove** [comment]
That's OK!

**Pull request merged**
